### PR TITLE
chore(sdk): Remove v2 components fork, use v1 instead. 

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Comment by Ark-kun** [comment]
/lgtm
/hold
Hold so that @numerology can take a look.

File: sdk/python/kfp/components/_components.py [hunk 1]
```diff
@@ -483,6 +483,7 @@ def _resolve_command_line_and_paths(
     argument_serializer: Callable[[str], str] = serialize_value,
     input_uri_generator: Callable[[str], str] = _generate_input_uri,
     output_uri_generator: Callable[[str], str] = _generate_output_uri,
+    input_value_generator: Optional[Callable[[str], str]] = None,
```
Comment (numerology) [inline sdk/python/kfp/components/_components.py:486]: Is there a place where this is specified?

Comment (chensun) [reply sdk/python/kfp/components/_components.py:486]: Yes, only from v2.dsl:

https://github.com/kubeflow/pipelines/blob/2dd8de3f6f7663cc6ef9207d901dd243ae169d74/sdk/python/kfp/v2/dsl/component_bridge.py#L190-L198

**Comment by chensun** [comment]
The kubeflow-pipelines-tfx-python36 failure is irrelevant to this PR, and tracked by https://github.com/kubeflow/pipelines/issues/5049.

I'm going to merge this PR, so that I can rebase and update my other PRs.

**Pull request merged**